vldb-best/example2.tex

```diff
@@ -123,8 +123,8 @@ \subsection{Example recursive query}\label{sec:recursive-example}
 recursive query; given an input relation \code{E} it will produce
 its transitive closure \code{R} as output.
 
-Now we use semina\"ive evaluation to rewrite the circuit (to save
-space we omit the indices from $\pi$ and $\sigma$):
+Now we use cycle rule to convert this circuit to seminaive evaluation
```

Comment:
```suggestion
Now we use cycle rule to convert this circuit to semi-naive evaluation
```

This is just for consistency, since it is spelled with a hyphen later.